spec/board_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'board'
+require 'game_errors'
+
+describe Board do
+
+  subject { Board.new }
+
+  describe "#check_win" do
+    shared_examples "win_example" do |parameter|
+      it "Slots #{parameter} return true" do
+        subject.reset
+        subject.instance_variable_set(:@slots, parameter)
```

Comment:
What about an idea to instead of using `instance_variable_set ` you use `fill_in_slot` method in order to set up a winning combination?

```
subject.fill_in_slot('a1', 'X')
subject.fill_in_slot('a2', 'X')
subject.fill_in_slot('a13, 'X')
```

In that case  even a spec description will be more meaningful. Instead of 

```
Slots [0, 1, 0, 0, 1, 0, 0, 1, 0] return true
```

we could read:

```
Slots a1, a2, a3 return true
```